### PR TITLE
WS - updating readmes.

### PR DESCRIPTION
**Pull request by idibuseba**
Updating some basic documentation notes.

### CODE REVIEW

**Review by benidibu — APPROVED** [review]

**Comment by yasikovsky** [comment]
@idibuseba I don't think that disclaimer will clear anything up, as the problem that we've had was because the client used a very long URL with a GET method for a data-heavy request, where as they should use the POST method with URL-encoded `data` and `hash` parameters.

**Comment by idibuseba** [comment]
Sorry @yasikovsky if I missed anything but my understanding of [your original comment](https://compono.atlassian.net/browse/IPE-332?focusedCommentId=43156) was that the issue was about the `application/x-www-form-urlencoded`, not the GET/POST method itself.
I was also able to replicate the issue they had and I was using POST (not GET) - so it seemed to me it was the header that ultimately fixed this. We do specify in the documentation already that the requests should be POST.

**Comment by yasikovsky** [comment]
@idibuseba sorry, let me rephrase the requirement: 

when using the POST method please send the parameters as form parameters, not as URL parameters.

**Review by yasikovsky — APPROVED** [review]
